src/query/ee/tests/it/ddl_basic/ddl_basic_01_create_udf.rs

```diff
@@ -0,0 +1,152 @@
+use arrow_udf_wasm::build::*;
+use databend_common_base::base::tokio;
+use databend_common_compress::CompressAlgorithm;
+use databend_common_compress::CompressCodec;
+use databend_common_exception::Result;
+use databend_common_storage::DataOperator;
+use databend_enterprise_query::test_kits::context::EESetup;
+use databend_query::test_kits::*;
+use opendal::Operator;
+use tokio::time::sleep;
+use tokio::time::Duration;
+use uuid::Uuid;
+
+#[tokio::test(flavor = "multi_thread")]
+async fn test_udf_js_gcd() -> Result<()> {
+    let fixture = TestFixture::setup_with_custom(EESetup::new()).await?;
+
+    // Create table
+    fixture
+        .execute_command_v2(r#"CREATE FUNCTION gcd (INT, INT) RETURNS BIGINT LANGUAGE javascript HANDLER = 'gcd_js' AS $$
+                export function gcd_js(a, b) {
+                    while (b != 0) {
+                        let t = b;
+                        b = a % b;
+                        a = t;
+                    }
+                    return a;
+                }
+            $$
+            "#
+        ).await?;
+
+    let res = fixture
+        .execute_command_v2(
+            r#"
+            select 
+                number, gcd(number * 3, number * 6) 
+            from 
+                numbers(5) 
+            where number > 0 order by 1;        
+            "#,
+        )
+        .await?;
+
+    log::info!("Response {:#?}", res);
+
+    Ok(())
```

Comment:
BTW we could use SQL logic tests in `03_0013_select_udf.test`, thus we don't need the log

Reply:
- Sure, for the final merger, the plan is to remove the unit test from the EE and convert it into SQL logic tests.

- Initially, for the development I used the approach of using the standalone server setup using the script `./scripts/ci/deploy/databend-query-standalone.sh`. However, tracing was somewhat challenging with this method. As an alternative, launching the unit test binary was found to be quite fast compared to launching the standalone servers. And I need to learn the tracing methodology `databend-common-tracing` used in this platform.